src/main/java/org/spongepowered/api/world/schematic/PaletteTypes.java

```diff
@@ -24,22 +24,14 @@
  */
 package org.spongepowered.api.world.schematic;
 
+import org.spongepowered.api.block.BlockState;
 import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+import org.spongepowered.api.world.biome.BiomeType;
 
-public class BlockPaletteTypes {
-
-    /**
-     * The global palette containing a mapping of all block states to ids.
-     */
-    public static final BlockPaletteType GLOBAL = DummyObjectProvider.createFor(BlockPaletteType.class, "GLOBAL");
-    /**
-     * A local palette containing only a subset of the global palette.
-     */
-    public static final BlockPaletteType LOCAL = DummyObjectProvider.createFor(BlockPaletteType.class, "LOCAL");
-
-    // Suppress default constructor to ensure non-instantiability.
-    private BlockPaletteTypes() {
-        throw new AssertionError("You should not be attempting to instantiate this class.");
-    }
+public class PaletteTypes {
```

Comment:
Missing default constructor with assertion error.

Reply:
Should be final class.